packages/cli/src/cmds/validator/voluntaryExit.ts

```diff
@@ -36,6 +37,12 @@ If no `pubkeys` are provided, it will exit all validators that have been importe
       command: "validator voluntary-exit --network goerli --pubkeys 0xF00",
       description: "Perform a voluntary exit for the validator who has a public key 0xF00",
     },
+    {
+      command:
+        "validator voluntary-exit --network goerli --externalSigner.url=http://signer:9000 --externalSigner.pubkeys 0xF00",
```

Comment:
This would require to provide all public keys if you'd want to exit all validators. Could use the following example
```suggestion
        "validator voluntary-exit --network goerli --externalSigner.url http://signer:9000 --externalSigner.fetch --pubkeys 0xF00",
```
I leave it up to you what you prefer, both works. Note, I also removed `=` from `externalSigner.url` for consistency